locales/nn_NO/LC_MESSAGES/duckduckgo.po

```diff
@@ -4215,23 +4215,23 @@ msgstr ""
 
 msgctxt "feedback form"
 msgid "We use feedback like this to improve DuckDuckGo. It really helps."
-msgstr ""
+msgstr "Vi brukar tilbakemeldingar som dette til å forbetra DuckDuckGo. Det hjelp oss verkeleg."
 
 msgctxt "new user poll"
 msgid "We're honored to have you on the Duck Side"
 msgstr "Vi er kjenner oss æra over å ha deg på «The Duck Side»"
 
 msgctxt "SERP footer content"
 msgid "We've got some more links for you!"
-msgstr ""
+msgstr "Vi har fleire lenkjer til deg"
 
 #. Instant Answer Tab Name
 msgid "Weather"
 msgstr "Vêr"
 
 msgctxt "feedback form"
 msgid "Web Links"
-msgstr "Nettlenker"
+msgstr "Nettlenkjer"
```

Comment:
For consistency. We shouldn't mix the two alternative spellings of this word ("lenkjer" was used more in the existing translations, and I think it sounds better)